README.md

```diff
@@ -117,7 +117,7 @@ Add an entry and define if it is a single or reoccurring payment. The following
 		TransactionCode:   "22",
 		IDNumber:          "#123456",
 		IndividualName:    "Wade Arnold",
-		PaymentType: 		"R"})
```

Comment:
I can add these helper functions for PaymentType back in after we merge this pull request. If you can leave PaymentType and ReceivingCompany in your pull request that would make it easier.

Reply:
This one should be ok